.github/dependabot.yml

```diff
@@ -1,17 +1,22 @@
 version: 2
 updates:
-- package-ecosystem: maven
-  directory: "/"
-  schedule:
-    interval: daily
-    timezone: Europe/Berlin
-  ignore:
-  - dependency-name: org.antipathy:mvn-scalafmt_2.12
-    versions:
-    - ">= 0.9_1.a, < 0.9_1.1000004"
-  - dependency-name: org.antipathy:mvn-scalafmt_2.12
-    versions:
-    - ">= 1.0.a, < 1.1"
-  - dependency-name: com.mycila:license-maven-plugin
-    versions:
-    - "4.0"
+  - package-ecosystem: maven
+    directory: "/"
+    schedule:
+      interval: daily
+      timezone: Europe/Berlin
+    ignore:
+      - dependency-name: org.antipathy:mvn-scalafmt_2.12
+        versions:
+          - ">= 0.9_1.a, < 0.9_1.1000004"
+      - dependency-name: org.antipathy:mvn-scalafmt_2.12
+        versions:
+          - ">= 1.0.a, < 1.1"
+      - dependency-name: com.mycila:license-maven-plugin
+        versions:
+          - "4.0"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
❓ For the `maven` ecosystem the schedule is set to a specific timezone. I'm not sure what the benefit is, but should we do the same for the GHA ecosystems?

Reply:
It ha to do with the time dependabot start analysing. I'm not sure what the default start time is, but it really doesn't matter. It runs once a day which is fine. I don't care what time it starts.